src/Raven.Server/Documents/CountersStorage.cs

```diff
@@ -602,89 +933,170 @@ public void DeleteCountersForDocument(DocumentsOperationContext context, string
             if (table.NumberOfEntries == 0)
                 return;
 
-            using (GetCounterPartialKey(context, documentId, out var keyPrefix))
+            using (DocumentIdWorker.GetLowerIdSliceAndStorageKey(context, documentId, out Slice lowerId, out _))
             {
-                table.DeleteByPrimaryKeyPrefix(keyPrefix);
+                //todo update Count of Counters
+                table.DeleteByKey(lowerId /*, holder => _countersCount -= GetData(context, ref holder.Reader).Count -1*/); 
```

Comment:
Remove this, we won't track the number of counters for now

Reply:
done